librosa/display.py

```diff
@@ -2461,6 +2504,186 @@ def waveshow(
     return adaptor
 
 
+def wavebars(
+    y: np.ndarray,
+    *,
+    sr: float = 22050,
+    n_bars: int = 100,
+    gap_ratio: float = 0.4,
+    rounding_ratio: float = 0.5,
+    axis: Optional[str] = "time",
+    offset: float = 0.0,
+    invert: bool = False,
+    invert_color: Union[str, tuple, None] = None,
+    transpose: bool = False,
+    label: Optional[str] = None,
+    ax: Optional[mplaxes.Axes] = None,
+    **patch_kwargs: Any,
+) -> mcollections.PatchCollection:
+    """Visualize a waveform as a series of bars representing the amplitude envelope.
+
+    This visualization is appropriate for displaying a simplified view of the
+    signal, and is best suited for small figures where simplicity is desired.
+
+    Parameters
+    ----------
+    y : np.ndarray [shape=(n,) or (2,n)]
```

Comment:
It would be good to explain here how to interpret the wavebar of a stereo y (i.e., upper vs lower halves)